process-engine/process_repository
# Feature/fix remove process

## What did you change?

The internal process has been added back, due to an accidental removal.

## How can others test the changes?

Check out process_engine_meta (branch 'feature/demo-test')

```shell
meta git update
meta exec --include-only=process_repository "git checkout feature/fix_remove_process"
meta exec --include-only=demo "git checkout feature/versioning"

npm install

meta exec --exclude=demo "gulp build"
meta exec --include-only=demo "npm run build"

cd demo
npm start
```

Go to ProcessManager via the menu. The remove process (via selection of a ProcessDef and then clicking the "bin"-Icon) can be started with the processDefId forwarded to the startToken. It will result in a wizard-like removal process of the ProcessDef at hand.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
